contingency/contingency-dsl/src/main/groovy/com/powsybl/contingency/dsl/ContingencyDslLoader.groovy

```diff
@@ -110,6 +111,70 @@ class ContingencyDslLoader extends DslLoader {
         }
     }
 
+    Map<String, Set<String>> loadNotFoundElements(Network network) {
+        loadNotFoundElements(network, null)
+    }
+
+    Map<String, Set<String>> loadNotFoundElements(Network network, ContingencyDslObserver observer) {
```

Comment:
Could you add a unit test to show it is really working ?
Maybe also we can reduce code duplication with the load method.

Reply:
done